test/package/test_package_fx.py

```diff
@@ -116,6 +128,33 @@ def test_package_fx_with_imports(self):
         packaged_dependency = pi.import_module("package_a.subpackage")
         self.assertTrue(packaged_dependency is not package_a.subpackage)
 
+    def test_package_fx_custom_tracer(self):
+        """
+        Tests that if a GraphModule containing a Graph
+        produced using a custom Tracer is packaged, is it unpackaged
+        using the same custom Tracer.
+        """
+        my_mod = MyModule()
+        tracer = CustomTracer()
+        graph = tracer.trace(my_mod)
+
+        self.assertIs(graph.tracer, tracer)
+
+        gm = GraphModule(my_mod, graph)
+
+        f = BytesIO()
+        with PackageExporter(f, verbose=False) as pe:
+            pe.extern(["io", "sys"])
+            pe.save_pickle("model", "model.pkl", gm)
+
+        f.seek(0)
+        pi = PackageImporter(f)
+        loaded = pi.load_pickle("model", "model.pkl")
+
+        # TODO: How to check that loaded uses CustomTracer (rather, packaged
+        # CustomTracer)?
+        # self.assertIsInstance(loaded.graph.tracer, CustomTracer)
```

Comment:
So this check obviously doesn't work because the loaded `CustomTracer` is a distinct class with the same name. What can I do instead?

Reply:
stash an attribute on the custom tracer that doesn't exist on normal tracers and make sure that it matches?